### PR TITLE
Renamed AuthenticationBaseViewModel

### PR DESCRIPTION
**Pull request by kamdaou-amalitech**
Renamed AuthenticationBaseViewModel

### CODE REVIEW

**Review by esthcarelle — APPROVED** [review]

File: core_ui/src/main/java/com/amalitech/core_ui/util/BaseViewModel.kt [hunk 1]
```diff
@@ -7,25 +7,25 @@ import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.flow.update
 
-open class AuthenticationBaseViewModel<T> : ViewModel() {
-    protected val baseResult: MutableStateFlow<UiState<T>?> =
+open class BaseViewModel<T> : ViewModel() {
+    protected val privateBaseResult: MutableStateFlow<UiState<T>?> =
         MutableStateFlow(null)
-    val publicBaseResult = baseResult.asStateFlow()
+    val baseResult = privateBaseResult.asStateFlow()
```
Comment (bensek) [inline core_ui/src/main/java/com/amalitech/core_ui/util/BaseViewModel.kt:13]: Rename privateBaseResult to _uiStateFlow and baseResult to uiStateFlow.

**Review by bensek — CHANGES_REQUESTED** [review]

**Review by bensek — APPROVED** [review]

**Comment by esthcarelle** [comment]
@kamdaou-amalitech  conflicts

**Comment by kamdaou-amalitech** [comment]
@esthcarelle resolved

**Comment by esthcarelle** [comment]
conflicts

**Comment by esthcarelle** [comment]
@kamdaou-amalitech conflicts please

**Pull request merged**